tests/test_files.py

```diff
@@ -1,223 +1,89 @@
-
-    assert is_ignored("ignored")
-    assert is_ignored("ignored/main.py")
-
-    assert is_ignored("src/ignored")
-    assert is_ignored("src/ignored/main.py")
-
-    assert is_ignored("../ignored")
-    assert is_ignored("../ignored/main.py")
-
-    assert is_ignored("../sub/ignored")
-    assert is_ignored("../sub/ignored/main.py")
-
-    assert not is_ignored("../../ignored")
-    assert not is_ignored("../../ignored/main.py")
-
-
-def test_ignore_git_in_working_parent_directory(
-    tmp_path: pathlib.Path, monkeypatch: pytest.MonkeyPatch
-) -> None:
-    monkeypatch.chdir(tmp_path)
-
-    (tmp_path / "sub").mkdir()
-    (tmp_path / "sub" / ".git").mkdir()
-    (tmp_path / "sub" / ".gitignore").write_text("ignored")
-
-    assert not is_ignored("ignored")
-    assert not is_ignored("ignored/main.py")
-
-    assert is_ignored("sub/ignored")
-    assert is_ignored("sub/ignored/main.py")
-
-    assert is_ignored("sub/src/ignored")
-    assert is_ignored("sub/src/ignored/main.py")
-
-    assert not is_ignored("sub/../ignored")
-    assert not is_ignored("sub/../ignored/main.py")
-
-
-def test_ignore_git_subdirectory_pattern(
-    tmp_path: pathlib.Path, monkeypatch: pytest.MonkeyPatch
-) -> None:
-    monkeypatch.chdir(tmp_path)
-
-    (tmp_path / ".git").mkdir()
-    (tmp_path / ".gitignore").write_text("sub/ignored")
-
-    (tmp_path / "sub").mkdir()
-
-    assert not is_ignored("sub")
-    assert not is_ignored("sub/main.py")
-
-    assert is_ignored("sub/ignored")
-    assert is_ignored("sub/ignored/main.py")
-
-
-def test_ignore_git_symlink_recursive(
-    tmp_path: pathlib.Path, monkeypatch: pytest.MonkeyPatch
-) -> None:
-    monkeypatch.chdir(tmp_path)
-
-    (tmp_path / ".git").mkdir()
-    (tmp_path / ".gitignore").write_text("ignored")
-
-    (tmp_path / "dir").mkdir()
-    (tmp_path / "dir" / "link").symlink_to(tmp_path / "dir")
-
-    assert not is_ignored("dir")
-    assert not is_ignored("dir/link")
-    assert not is_ignored("dir/link/link")
-
-    assert is_ignored("dir/ignored")
-    assert is_ignored("dir/link/ignored")
-    assert is_ignored("dir/link/link/ignored")
-
-
-def test_ignore_git_symlink_outside_repo(
-    tmp_path: pathlib.Path, monkeypatch: pytest.MonkeyPatch
-) -> None:
-    monkeypatch.chdir(tmp_path)
-
-    (tmp_path / "repo" / ".git").mkdir(parents=True)
-    (tmp_path / "repo" / ".gitignore").write_text("link")
-
-    (tmp_path / "link").mkdir()
-    (tmp_path / "repo" / "link").symlink_to(tmp_path / "link")
-
-    assert not is_ignored("link")
-    assert not is_ignored("link/main.py")
-    assert is_ignored("repo/link")
-    assert is_ignored("repo/link/main.py")
-
-
-def test_ignore_symlink_circular(
-    tmp_path: pathlib.Path, monkeypatch: pytest.MonkeyPatch
-) -> None:
-    monkeypatch.chdir(tmp_path)
-
-    (tmp_path / "link1").symlink_to(tmp_path / "link2")
-    (tmp_path / "link2").symlink_to(tmp_path / "link1")
-
-    assert not is_ignored("link1")
-    assert not is_ignored("link2")
```

Comment:
Would it be possible to restore these tests so that we can see whether the current no-pyproject.toml behaviour has changed?